imagersite/imager_images/templates/imager_images/album_gallery.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'imagersite/base.html' %}
+{% load thumbnail %}
+{%  block body %}
+    <h3>{{ user.username }} Albums</h3>
+
+    <ul class="thumbnail">
+        {% for album in albums %}
+        <li>
+            {% thumbnail album.cover.photo "100x100" crop="center" as im %}
+                <img src="{{ im.url }}" width="{{ im.width }}" height="{{ im.height }}">
+            {% endthumbnail %}
+            <a href="{% url 'imager_images:album' %}">{{album.title}}</a>
```

Comment:
`<a href="{% url 'imager_images:album' %}">{{album.title}}</a>` should be `<a href="{% url 'imager_images:album' 
 album.id %}">{{album.title}}</a>`